elliottlib/cli/__main__.py

```diff
@@ -133,6 +135,10 @@ def get(ctx, runtime, default_advisory_type, details, id_only, as_json, advisory
         click.echo(advisory)
         return
 
+    if greenwave:
+        click.echo(elliottlib.errata.get_advisory_greenwave(advisory))
```

Comment:
This would completely change the functionality of the `elliott get` command, which originally means `Get details about a specific advisory from the Errata Tool`. If it only displays the greenwave test result, it is better to have a separate command. Otherwise this info can be added on top of the current advisory data.